docs/README.md

```diff
@@ -22,10 +15,12 @@ A Javascript SDK for receiving updates from the OpenSea Stream API - pushed over
 
 This is a best effort delivery messaging system. Messages that are not received due to connection errors will not be re-sent. Messages may be delievered out of order. This SDK is offered as a beta experience as we work with developers in the ecosystem to make this a more robust and reliable system.
 
-# Setup
+# Installation
+
+We recommend switching to Node.js version 16 to make sure common crypto dependencies work. Our minimum supported version is 16.11.0.
 
-Run `nvm use`  
-And then `npm install`
+- If you have Node Version Manager, run `nvm use` to ensure a compatible version is being used.
+- Install this package with `yarn install --save @opensea/stream-api`
```

Comment:
are we supposed to have two sets of readme?

Reply:
This is a frustrating side affect of me wanting to test run an integration with our docs platform to automatically sync the markdown file as generated docs. 

The Readme team is actually working on a fix for this issue so we won't have to do this. They iterate pretty quickly so hopefully this comes soon. Either way, will work through a way around keeping both soon